### PR TITLE
link_pseudo: Normalize keyboard interaction #932

### PR DESCRIPTION
**Pull request by hcodes**

### CODE REVIEW

**Comment by hcodes** [comment]
resolves #932
old #1029

File: common.blocks/link/_pseudo/link_pseudo.js [hunk 1]
```diff
@@ -10,9 +10,29 @@ modules.define('link', function(provide, Link) {
...
+                this.unbindFrom('control', 'keydown', this._onKeyDown);
+            }
+        }
+    },
     _onPointerClick : function(e) {
         e.preventDefault();
+
         this.__base.apply(this, arguments);
+    },
+    _onKeyDown : function(e) {
```
Comment (dfilatov) [inline common.blocks/link/_pseudo/link_pseudo.js:34]: Add empty line between methods.

File: common.blocks/link/_pseudo/link_pseudo.js [hunk 2]
```diff
@@ -10,9 +10,29 @@ modules.define('link', function(provide, Link) {
...
+        if(e.keyCode === keyCodes.ENTER) {
```
Comment (dfilatov) [inline common.blocks/link/_pseudo/link_pseudo.js]: use `&&` instead of `if`

**Comment by dfilatov** [comment]
squash, rebase and merge

**Comment by hcodes** [comment]
@dfilatov Fixed.

**Pull request merged**